### ISSUE TITLE
figure out multi-label multi-class with sklearn

### ISSUE THREAD

**Issue by mrivantan** [mrivantan/ga-capstone#2]
https://scikit-learn.org/dev/auto_examples/miscellaneous/plot_multilabel.html

https://scikit-learn.org/stable/modules/multiclass.html

https://scikit-learn.org/stable/tutorial/text_analytics/working_with_text_data.html
